### PR TITLE
Feat(imprint and data security link): add imprint and data security link

### PR DESCRIPTION
**Pull request by JonathanXITASO**
# Description

Added clickable 'Imprint' and 'Data Security' links and opens them in a new tab.
Link urls are configurable through env-vars.
Added simple component test for the external clickable link
Restructured the MainMenu to be grouped into 1. features, 2. legal&information, 3. authentication

Fixes # (MNES-128)

## Type of change

-   [x] New feature (non-breaking change which adds functionality)

# Checklist:

-   [x] I have performed a self-review of my code
-   [x] I have commented my code, particularly in hard-to-understand areas
-   [x] I have made corresponding changes to the documentation
-   [x] My changes generate no new warnings
-   [x] I have added tests that prove my fix or my feature works
-   [x] New and existing tests pass locally with my changes
-   [x] My changes contain no console logs

### CODE REVIEW

File: src/layout/menu/ExternalLink.spec.tsx [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+import { messages } from 'lib/i18n/localization';
+
+describe('ExternalLink', () => {
+    it('should render the component', async () => {
+        CustomRenderReactIntl(<ExternalLink href="https://xitaso.com/" descriptor={messages.mnestix.imprint} />);
+        const component = screen.getByTestId('external-link');
+        expect(component).toBeInTheDocument();
+    });
+
+    it('should open link in new tab natively', async () => {
```
Comment (JonathanXITASO) [inline src/layout/menu/ExternalLink.spec.tsx:14]: I could not find a way to actually test, if a new tab is opened.
The `target="_blank"` attribute will use the native browser feature to open in a new tab and will not use `window.open`, so we cannot mock it

File: src/layout/menu/MainMenu.tsx [hunk 2]
```diff
@@ -58,7 +59,18 @@ export default function MainMenu() {
...
+    const copyrightString = `Copyright © ${new Date().getFullYear()} XITASO GmbH`;
     const versionString = 'Version ' + packageJson.version;
+    const imprintString = env.IMPRINT_URL;
+    const dataPrivacyString = env.DATA_PRIVACY_URL;
+
+    const getAuthName = () => {
+        const user = auth?.getAccount()?.user;
+        if (!user) return;
+        if (user.email) return user.email;
+        if (user.name) return user.name;
```
Comment (JonathanXITASO) [inline src/layout/menu/MainMenu.tsx:71]: This will show the username for keycloak logins

**Comment by JonathanXITASO** [comment]
Should xitaso be set as the default value in the compose file as well as in `.env.local`?

File: src/layout/menu/MainMenu.tsx [hunk 3]
```diff
@@ -58,7 +59,18 @@ export default function MainMenu() {
     const auth = useAuth();
     const env = useEnv();
     const useAuthentication = env.AUTHENTICATION_FEATURE_FLAG;
+    const copyrightString = `Copyright © ${new Date().getFullYear()} XITASO GmbH`;
```
Comment (NilsXitaso) [inline src/layout/menu/MainMenu.tsx:62]: Do we use XITASO here our should we use Eclipse ?

**Review by NilsXitaso — APPROVED** [review]

**Pull request merged**